include/cinder/GeomIo.h

```diff
@@ -210,8 +210,12 @@ class Rect : public Source {
 	Rect( const Rectf &r );
 
 	Rect&		rect( const Rectf &r );
+	//! Enables default colors. Disabled by default.
+	Rect&		colors();
```

Comment:
I prefer enable with flag. I didn't notice that in ico. Was trying to match cube.

On Fri, Mar 20, 2015 at 2:43 PM, Richard Eakin <notifications@github.com>
wrote:

>> @@ -210,8 +210,12 @@ class Rect : public Source {
>>  	Rect( const Rectf &r );
>>  
>>  	Rect&		rect( const Rectf &r );
>> +	//! Enables default colors. Disabled by default.
>> +	Rect&		colors();
> It looks like we handle this differently in different places, however I think the best way is to pass a bool in, i.e. `colors( bool enable = true )`, instead of having hte addition `disableColors()` method. This is how it is done in [Icosehedron](https://github.com/cinder/Cinder/blob/glNext/include/cinder/GeomIo.h#L269), but obviously not in [Cube](https://github.com/cinder/Cinder/blob/glNext/include/cinder/GeomIo.h#L237-L241).
> Above all, I think consistency is important so we should probably pick one or the other and make them all match.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cinder/Cinder/pull/758/files#r26879008